planetscale/keyspaces.go

```diff
@@ -314,3 +336,17 @@ func (s *keyspacesService) RolloutStatus(ctx context.Context, rolloutReq *Keyspa
 
 	return rolloutStatusResponse, nil
 }
+
+func (s *keyspacesService) UpdateSettings(ctx context.Context, updateReq *UpdateKeyspaceSettingsRequest) (*Keyspace, error) {
+	req, err := s.client.newRequest(http.MethodPatch, keyspaceAPIPath(updateReq.Organization, updateReq.Database, updateReq.Branch, updateReq.Keyspace), updateReq)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating http request")
```

Comment:
@notfelineit Removing the usage of `errors.Wrap` in #250, so we should use `fmt.Errorf` here with a `%w` for the error wrapping.

Reply:
@notfelineit Took the liberty of updating it already here :smile:.

Reply:
Thank you @dbussink ! 😄